0019-refactor-image-operations/README.md

```diff
@@ -0,0 +1,315 @@
+
+### Goals
+
+<!--
+List the specific goals of the ZEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+- Improve reliability of container image pushing and pulling.
+- Introduce shared cache between Zarf package OCI operations and Zarf image OCI operations.
+
+### Non-Goals
+
+<!--
+What is out of scope for this ZEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+- Remove the `zarf tools crane` command which provides users with a CLI to interact with container registries. Crane will still be a dependency of Zarf because of this. Additionally, the Syft requires Crane objects to generate SBOMs. 
+
+## Proposal
+
+<!--
+This is where you explain the specifics of the proposal. Provide enough detail
+for reviewers to clearly understand what you're proposing, but avoid including
+too many specifics like API designs or implementation details. Focus on the
+desired outcome and how success will be measured. The "Design Details" section
+below is for the real nitty-gritty.
+-->
+
+Use the oras-go library to replace Crane for image pull and image push operations. 
+
+Image pull and push operations will respect the global `--oci-concurrency` flag to determine how many layers to push or pull concurrently. This flag is currently used for Zarf package OCI operations, but not respected during image operations with Crane. The default `--oci-concurrency` flag value will increase to six. The default in oras-go and Zarf is currently three, but this number is conservative. [Skopeo](https://github.com/containers/skopeo), for instance, has a default of six. Crane pushes and pulls all layers concurrently always.
+
+The oras-go implementation will only pull one image at a time. The current implementation pulls up to ten images concurrently, while this may improve speed in certain cases, in many others, Zarf is over saturating the network and worsening reliability. If in the future Zarf would like to go back to concurrently pulling images, oras-go will handle this without issue. This can be seen by the [DoOrasPullConcurrent](https://github.com/zarf-dev/image-pull-experiments/blob/main/oras/main.go#L55) function created to test this behavior.
+
+A shared cache will be used for Zarf package OCI operations and Zarf image OCI operations, see issue [2033](https://github.com/zarf-dev/zarf/issues/2033) which requests this feature. 
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this ZEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+This ZEP proposes a technical refactoring and does not change the users flow. 
+
+### Risks and Mitigations
```

Comment:
It's ok to add to this as you experiment with the new implementations. Backwards compatibility and feature flagging help mitigate risk of behavior inconsistencies, but do they overly complicate the transition and get in the way of us making the UX better? All tradeoffs worth consideration.